port/common/j9nls.c

```diff
@@ -234,34 +232,30 @@ writeSyslog(struct OMRPortLibrary *portLibrary, uintptr_t flags, const char *for
 		return;
 	}
 
-	/* Attempt to write string to on-stack buffer */
-	COPY_VA_LIST(copyOfArgs, args);
-	numberWritten = portLibrary->str_vprintf(portLibrary, outputBuffer, sizeof(outputBuffer), format, copyOfArgs);
+	/* What is size of buffer required ? str_vprintf(..,NULL,..) result includes the null terminator */
+	uintptr_t bufferSize = portLibrary->str_vprintf(portLibrary, NULL, 0, format, args);
```

Comment:
You can not declare the variable here in c89 which is used on some of our platforms